records_mover/utils/concat_files.py

```diff
@@ -18,10 +20,23 @@ def readable(self) -> bool:
         return True
 
     def readall(self) -> bytes:
-        return b''.join(f.read(-1) for f in self._files)
+        out = b''
+        while self._files:
+            f = self._files.pop(0)
+            out = out + f.read()
+        # TODO write a test that forces me to write this code
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker